RHDEVDOCS 3306: Document running image build tasks as unprivileged bu… (#50364)
openshift/openshift-docs: cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc

:_content-type: ASSEMBLY
[id="running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines"]
= Running workloads and Buildah as user namespaces on {pipelines-title}
include::_attributes/common-attributes.adoc[]
:context: running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines

toc::[]

> Running {pipelines-shortname} as root user on a container can expose the host and container processes to other resources. You can reduce this type of exposure by using one of the following methods:

Reviewer: ```suggestion
Running {pipelines-shortname} as the root user on a container can expose the host and container processes to other resources. You can reduce this type of exposure by using one of the following methods:
```
- needs a definite article
- Would it be useful to mention that the `other resources` could be potentially malicious? It suppose it's implied with the current phrasing, but I think it wouldn't hurt to make it explicit. WDYT?

> * You can run workloads in a user namespace.

Reviewer: ```suggestion
* Run workloads in a user namespace.
```
- The lead-in sentence above already said `you can`. I think it's ok to use the imperative here to reduce the number of words.

> * You can run the workload as a root user on the container and make it secure on the host.

Reviewer: ```suggestion
* Run workloads as a root user on the container and make it secure on the host.
```
- The lead-in sentence above already said `you can`. I think it's ok to use the imperative here to reduce the number of words.
- The antecedent of "it" in `make it secure` is not clear. Is "it" the container? or the workload?
